README.md

```diff
@@ -93,43 +113,175 @@ This is a TypeScript class that represents a holder of Verifiable Credentials (V
 
 #### Usage
 
-To use the Holder class, you need to create an instance of it by passing in a signer configuration object. Here's an example:
+To use the Holder class, you need to create an instance of it by passing in a Holder Signer configuration object. Here's an example:
 
 ```typescript
-import { importJWK } from 'jose';
-import { Holder } from './holder';
-import { signerCallbackFn, veriferCallbackFn } from './test-utils/helpers';
-import { supportedAlgorithm } from './util';
-
-const privateKey = {
-  kty: 'EC',
-  x: 'rH7OlmHqdpNOR2P28S7uroxAGk1321Nsgxgp4x_Piew',
-  y: 'WGCOJmA7nTsXP9Az_mtNy0jT7mdMCmStTfSO4DjRsSg',
-  crv: 'P-256',
-  d: '9Ie2xvzUdQBGCjT9ktsZYGzwG4hOWea-zvCQSQSWJxk',
+import { KeyBindingVerifier, Signer, decodeJWT } from '@meeco/sd-jwt';
+import { JWK, JWTHeaderParameters, JWTPayload, KeyLike, SignJWT, importJWK, jwtVerify } from 'jose';
+import { Holder, SignerConfig, supportedAlgorithm } from 'src';
+
+const signerCallbackFn = function (privateKey: Uint8Array | KeyLike): Signer {
+  return (protectedHeader: JWTHeaderParameters, payload: JWTPayload): Promise<string> => {
+    return new SignJWT(payload).setProtectedHeader(protectedHeader).sign(privateKey);
+  };
 };
 
-const pk = await importJWK(privateKey);
-let signer: SignerConfig = {
-  alg: supportedAlgorithm.EdDSA,
-  callback: signerCallbackFn(pk),
-};
-const holder = new Holder(signer);
+async function main() {
+  const _publicJwk = {
+    kty: 'EC',
+    x: 'rH7OlmHqdpNOR2P28S7uroxAGk1321Nsgxgp4x_Piew',
+    y: 'WGCOJmA7nTsXP9Az_mtNy0jT7mdMCmStTfSO4DjRsSg',
+    crv: 'P-256',
+  };
+  const privateKey = {
+    kty: 'EC',
+    x: 'rH7OlmHqdpNOR2P28S7uroxAGk1321Nsgxgp4x_Piew',
+    y: 'WGCOJmA7nTsXP9Az_mtNy0jT7mdMCmStTfSO4DjRsSg',
+    crv: 'P-256',
+    d: '9Ie2xvzUdQBGCjT9ktsZYGzwG4hOWea-zvCQSQSWJxk',
+  };
+
+  const pk = await importJWK(privateKey);
+
+  const signer: SignerConfig = {
+    alg: supportedAlgorithm.ES256,
+    callback: signerCallbackFn(pk),
+  };
+  const holder = new Holder(signer);
+}
+
+main();
 ```
 
 #### presentVerifiableCredentialSDJWT
```

Comment:
I would consider using `VC` instead of `VerifiableCredential`. I think we all understand what does it stand for. Makes it a little shorter. `presentVCSDJWT`.

Reply:
works, i will change it.